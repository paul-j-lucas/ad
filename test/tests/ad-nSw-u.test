ad | -c always -n -Sw -u always | utf8.bin | | 0
