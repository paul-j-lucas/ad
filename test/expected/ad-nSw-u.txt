[32m[K0000000000000000[m[K[36m[K:[m[K [41;1m[KE280[m[K [41;1m[K9C48[m[K [41;1m[K656C[m[K [41;1m[K6C6F[m[K  [41;1m[K2C20[m[K [41;1m[K776F[m[K [41;1m[K726C[m[K [41;1m[K6421[m[K  [41;1m[K‚Äú‚ñ°‚ñ°Hello, world![m[K
[32m[K0000000000000010[m[K[36m[K:[m[K [41;1m[KE280[m[K [41;1m[K9D0A[m[K [41;1m[K3132[m[K [41;1m[K3334[m[K  [41;1m[K3536[m[K [41;1m[K3738[m[K [41;1m[K39E2[m[K [41;1m[K989E[m[K  [41;1m[K‚Äù‚ñ°‚ñ°.123456789‚òû‚ñ°‚ñ°[m[K
[32m[K0000000000000020[m[K[36m[K:[m[K [41;1m[K0A31[m[K [41;1m[K3233[m[K [41;1m[K3435[m[K [41;1m[K3637[m[K  [41;1m[K3839[m[K [41;1m[K3031[m[K [41;1m[K3233[m[K [41;1m[KE298[m[K  [41;1m[K.1234567890123‚òû‚ñ°[m[K
[32m[K0000000000000030[m[K[36m[K:[m[K [41;1m[K9E0A[m[K [41;1m[K3132[m[K [41;1m[K3334[m[K [41;1m[K3536[m[K  [41;1m[K3738[m[K [41;1m[K3930[m[K [41;1m[K3132[m[K [41;1m[K33E2[m[K  [41;1m[K‚ñ°.1234567890123‚òû[m[K
[32m[K0000000000000040[m[K[36m[K:[m[K [41;1m[K989E[m[K [41;1m[K340A[m[K [41;1m[K[m[KE298 31E2  3132 0A              [41;1m[K‚ñ°‚ñ°4.[m[K..1.12.
